servicetalk-grpc-netty/src/test/java/io/servicetalk/grpc/netty/GrpcProxyTunnelTest.java

```diff
@@ -89,7 +90,7 @@ class GrpcProxyTunnelTest {
                 .listenAndAwait((Greeter.BlockingGreeterService) (ctx, request) ->
                         HelloReply.newBuilder().setMessage(GREETING_PREFIX + request.getName()).build());
         client = GrpcClients.forAddress(serverHostAndPort(serverContext))
-                .initializeHttp(httpBuilder -> httpBuilder.proxyAddress(proxyAddress)
+                .initializeHttp(httpBuilder -> httpBuilder.proxyConfig(ProxyConfig.of(proxyAddress))
```

Comment:
I debated between `ProxyConfig.forAddress(proxyAddress)` and `ProxyConfig.of(proxyAddress)`. Went with `of` only because wanted to be consistent with similar static method on other interfaces/classes, like `HostAndPort`, `HttpRequestMethod`, etc. If `of` doesn't read well for this use-case, happy to rename it by your recommendations.